config/navigationLinks.js

```diff
@@ -418,6 +418,10 @@ const navigationLinks = [
                 name: "Delayed::Job",
                 link: "/ruby/integrations/delayed-job.html",
               },
+              {
+                name: "Garbage Collection Profiler",
```

Comment:
Should this say "Garbage Collection", without "Profiler"? That's the name used in the document title and in the integrations index.
```suggestion
                name: "Garbage Collection",
```